core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -872,6 +882,33 @@ protected void doRun() throws Exception {
     }
 
     /**
+     * Get the Jenkins {@link StartupType startup type}.   
+     * @return The Jenkins {@link StartupType startup type}.
+     * @since FIXME
+     */
+    public StartupType getStartupType() {
+        return startupType;
+    }
+
+    /**
+     * Get the URL path to the Install Wizard JavaScript.
+     * @return The URL path to the Install Wizard JavaScript.
+     * @since FIXME
+     */
+    public String getInstallWizardJS() {
```

Comment:
`@Restricted(NoExternalUse.class)`. (Or even get this out of `Jenkins` entirely, unless you are actually using it for Stapler binding.)